osc-dm-mesh-cli/src/cli.py

```diff
@@ -520,7 +520,6 @@ async def execute(xargs=None):
     # Used for the artifact generation
     products_parser.add_argument("--type", help="type of artifact being generated (service)")
     products_parser.add_argument("--service_host", help="Model to use for generating description (must align to vendor)")
```

Comment:
pretty sure the description for service host here is incorrect. It is not a model, but a URL. Suspect you copy-pasted the wrong description when making the param.

Reply:
changed description to match parameter